docs/v0.9/en/getting-started/installation/greptimedb-cluster.md

```diff
@@ -1,73 +1,150 @@
 
-If your Kubernetes cluster is ready(a 1.18 or higher version is required), you can use the following command to deploy the GreptimeDB cluster:
+We are using [GreptimeTeam/helm-charts](https://github.com/GreptimeTeam/helm-charts) to deploy the GreptimeDB cluster in Kubernetes.
+
+### Prerequisites
+
+- Kubernetes >= 1.18
+
+  :::tip Note
+
+  You can use the [kind](https://kind.sigs.k8s.io/docs/user/quick-start/) to create a local Kubernetes cluster for testing:
+
+  ```
+  kind create cluster
+  ```
+
+  :::
+
+- Install [`helm`](https://helm.sh/docs/intro/install/)
+
+- Install [`kubectl`](https://kubernetes.io/docs/tasks/tools/)
+
+### Step 1: Add Chart Repository
+
+You can add the chart repository with the following commands:
 
 ```
-./gtctl cluster create mycluster
+helm repo add greptime https://greptimeteam.github.io/helm-charts/
+helm repo update
 ```
 
-After the creation is completed, you can use the following command to connect the cluster with MySQL protocol:
+You can run the following command to see the charts:
 
 ```
-./gtctl cluster connect mycluster
+helm search repo greptime
 ```
 
-You also can use [Helm Charts](/user-guide/operations/deploy-on-kubernetes/overview.md) to deploy the cluster.
+### Step 2: Deploy the etcd cluster
+
+We recommend using the Bitnami etcd [chart](https://github.com/bitnami/charts/blob/main/bitnami/etcd/README.md) to deploy the etcd cluster:
+
+```
+helm upgrade \
+  --install etcd oci://registry-1.docker.io/bitnamicharts/etcd \
+  --set replicaCount=3 \
+  --set auth.rbac.create=false \
+  --set auth.rbac.token.enabled=false \
+  --create-namespace \
+  -n etcd-cluster
+```
 
 :::tip Note
 
-You can use the [kind](https://kind.sigs.k8s.io/docs/user/quick-start/) to create the Kubernetes:
+You can skip the step if you already have an available etcd cluster in Kubernetes.
+
+:::
+
+### Step 3: Deploy greptimedb-operator
+
+The [greptimedb-operator](https://github.com/GrepTimeTeam/greptimedb-operator) is a Kubernetes operator that manages the GreptimeDB cluster. We can install greptimedb-operator in the `greptimedb-admin` namespace:
 
+``` 
+helm upgrade \
+  --install \
+  --create-namespace \
+  greptimedb-operator greptime/greptimedb-operator \
+  -n greptimedb-admin
 ```
-kind create cluster
+
+### Step 4: Deploy the GreptimeDB cluster
+
+We can deploy the cluster by using the default storage in the `default` namespace:
+
+```
+helm upgrade \
+  --install mycluster \
+  --set meta.etcdEndpoints=etcd.etcd-cluster.svc.cluster.local:2379 \
+  --set image.tag=<version>
+  greptime/greptimedb-cluster \
+  -n default
 ```
 
-:::
+### Step 5: Use the `kubectl port-forward` to access the cluster
 
-When the cluster is ready on your Kubernetes, you can use the following commands to expose all the service ports(make sure you already install [kubectl](https://kubernetes.io/docs/tasks/tools/)):
+To access the port `4000-4003` of the cluster, we can use the `kubectl port-forward`:
 
 ```
```

Comment:
The same question: could we add the version?

Reply:
It seems the `kubectl` URL doesn't support specifying the version. However, 
the URL here points to the installation guide instead of a specific version. `kubectl` is a stable tool, and users can always use the latest stable version of the release.